pkg/cache/sql/db/client_test.go

```diff
@@ -296,6 +296,96 @@ func TestQueryStrings(t *testing.T) {
 	}
 }
 
+func TestReadInt(t *testing.T) {
+	type testCase struct {
+		description string
+		test        func(t *testing.T)
+	}
+
+	var tests []testCase
+
+	testResult := 42
+	tests = append(tests, testCase{description: "One row, no errors", test: func(t *testing.T) {
+		c := SetupMockConnection(t)
```

Comment:
Lots of repetition here ... lines 309 through 312 and 320 can be done in one helper func that returns `(*MockRows, *Client)` all through this test. And then the tests are more readable because the setup is minimized.